### DISCUSSION TITLE
"Require alt text for images" toggle setting does not apply across devices

### DISCUSSION THREAD

**Discussion by douglaschu** [bluesky-social/atproto#1380] [General Discussion]
I toggled "require alt text" from the ios mobile app. This did not toggle the setting on my desktop browser. The setting seems to be stored in the device's LocalStorage cache.

**Comment by bnewbold** [comment]
Yeah, I think "require alt text" may intentionally be a device-local setting.

@pfrazee can you confirm?

**Reply by pfrazee** [reply]
Yes that's presently how it's implemented; might need to move to backend
